docs/maintainers/develop.md

```diff
@@ -0,0 +1,94 @@
+# Developing Quantum Mobile
+
+## Code style
+
+Code style is tested using pre-commit:
+
+```bash
+pip install pre-commit
+pre-commit run --all
+```
+
+## Updating the documentation
+
+```bash
+pip install tox
+# to build from scratch
+tox -e docs-clean
+# to start a live server (reloads automatically on changes to documentation)
+tox -e docs-live
+```
+
+## Preparing a release
+
+### Versions to check/update
+
+* [Vagrant](https://www.vagrantup.com/downloads.html)
+* Vagrant [vbguest plugin](https://github.com/dotless-de/vagrant-vbguest) (`vagrant plugin update`)
+* [bento/ubuntu-18.04](https://app.vagrantup.com/bento/boxes/ubuntu-18.04) Vagrant image (`vagrant box update`)
+* [Virtual Box](https://www.virtualbox.org/wiki/Downloads)
+* QM codes
+  * [Quantum Espresso](https://gitlab.com/QEF/q-e/tags)
+  * [cp2k](https://github.com/cp2k/cp2k/releases)
+  * [siesta](https://gitlab.com/siesta-project/siesta/-/releases)
+  * [yambo](https://github.com/yambo-code/yambo/wiki/Releases-(tar.gz-format))
+  * [fleur](https://www.flapw.de/master/downloads/)
+  * [wannier90](https://github.com/wannier-developers/wannier90/releases)
+  * [bigdft](https://gitlab.com/l_sim/bigdft-suite/-/releases)
+* [AiiDA packages on PyPI](https://pypi.org/search/?q=aiida&o=-created)
+* Quantum Mobile itself
+
+### Building the VM
+
+Follow the [vagrant build instructions](../developers/build-vagrant.md).
+
+:::{note}
+One step that is currently not automated is to double-click on the Desktop symbols to show the icons.
+:::
+
+### Releasing the VM
```

Comment:
@ltalirz I've changed this to be more in-line with your suggestion in https://github.com/marvel-nccr/quantum-mobile/pull/131#issuecomment-708280513

Reply:
thanks!